### ISSUE TITLE
Actividad 4: Division de los sets

### ISSUE THREAD

**Issue by andrescf** [IIC2613-Inteligencia-Artificial-2021-2/Syllabus#150]
Hola! En la actividad 4 se pide ordenar y luego dividir pero la funcion "train_test_split" tiene por defecto shuffle = True, lo que haria que el ordenar los datos antes no tenga importancia, pero tampoco se especifica si debemos cambiar ese parametro. Por lo que vi en las otras issues me imagino que deberiamos setearlo en False pero no estoy seguro.
Gracias!

**Comment by FeBalla** [comment]
Comprendo tu duda. Con respecto a eso puedo decirte un par de cosas:
* Lo fundamental, es que luego de ordenar los datos, ocupes el primer 70% para entrenamiento.
* Dicho lo anterior, no es necesario ocupar `train_test_split`, bastaría con indexar.
* Si logras hacerlo con `train_test_split`, ya sea modificando parámetros o no, es totalmente a criterio tuyo, aunque debes cumplir en cualquier caso el requisito del primer punto.

**Comment by andrescf** [comment]
Perfecto, me quedo mas claro. Muchas gracias!!
